main/res/values/strings_not_translatable.xml

```diff
@@ -85,7 +85,8 @@
 	</string>
 
     <!-- cache details -->
-    <string name="favorite_count" translatable="false">%1$d×</string>
+    <string name="favorite_count" translatable="false">%1$d</string>
+    <string name="favorite_count_percent" translatable="false">%1$d (%2$.0f\%)</string>
```

Comment:
I think we should add a '×' to make it consistent with the number without percentages.

Reply:
This is consistent with number without "x"